rbac/providers/ldap/initial_inbound_sync.py

```diff
@@ -93,7 +95,9 @@ def insert_to_db(data_dict, data_type):
             "sync_type": "initial",
             "timestamp": datetime.now().replace(tzinfo=timezone.utc).isoformat(),
             "provider_id": LDAP_DC,
+            "raw": entry_data,
         }
+        add_transaction(inbound_entry)
```

Comment:
This is adding transaction information prior to being loaded into the inbound queue.  I was under the impression that this sort of processing would be done post insertion into the inbound queue.  This seems to be in the wrong part of the flow as I thought this is supposed to go into the NEXT portion of the application.

Reply:
I am also confused about this. The queues are the handoff points between modules. inbound_sync should be publishing to this queue and ledger_sync should be subscribing.

Reply:
Please relocate this operation to the inbound processing step of ledger_sync. 
Ldap inbound enqueues the records, ledger_sync dequeues and processes them.

Reply:
resolved